test/test.js

```diff
@@ -28,37 +26,45 @@ describe('saving and loading works', function () {
     return chunk
   }
 
-  let regionPath = 'world/testRegion'
```

Comment:
Why ? This means no Independence between tests which will give weird results

Reply:
Why setting regionPath to const ? Because it was never modified. The dependence between test was already there.

Reply:
Ah no sorry I commented on the wrong line. I meant before -> before all

before will run before every test whereas beforeAll will run just once.

It's better to clean up stuff before every test

Reply:
In that case that would break the loading test that rely on the output of the saving test. I agree that we shouldn't have dependency between tests but this is out of the scope of this PR.

Reply:
That's the thing. What you describe is the previous behavior. In this PR you introduced a dependency between tests

Reply:
No, look at theses lines:
https://github.com/PrismarineJS/prismarine-world/pull/28/files#diff-c1129c8b045390789fa8ff62f2c6b4a9L44
https://github.com/PrismarineJS/prismarine-world/pull/28/files#diff-c1129c8b045390789fa8ff62f2c6b4a9L57
The loading test needs the world to be in the folder in order to load it. If you clean before/after every test then the world will be empty when the loading test is executed.
The dependency was already there. What I did was only changing mocha to jest to conform to other prismarine projects. In mocha, `before()` is the equivalent of jest's `beforeAll()` (both have a `beforeEach()` function but that would break the tests has the dependency is needed here) (sources: https://mochajs.org/#hooks https://jestjs.io/docs/en/api )

Reply:
Ah yeah got it you're right